openfl/component/aggregator/aggregator.py

```diff
@@ -667,6 +672,44 @@ def send_local_task_results(
 
             self._end_of_round_with_stragglers_check()
 
+    def get_memory_usage(self, round_number, metric_origin):
```

Comment:
maybe call this `get_host_memory_usage` ?

Reply:
I added details of process as well in get_memory_usage in the function.

Reply:
Makes sense, dropping my comment.